photon_installer/installer.py

```diff
@@ -297,6 +297,19 @@ def _add_defaults(self, install_config):
         if 'photon_docker_image' not in install_config:
             install_config['photon_docker_image'] = "photon:latest"
 
+        # if "repos" key not present in install_config or "repos=" provided by user through cmdline prioritize cmdline
+        if "repos" not in install_config or self.repo_paths != "/mnt/media/RPMS" :
```

Comment:
Please define "/mnt/media/RPMS" as a constant. Otherwise, changing the path value only in one place will unintentionally break the logic